app/classifier/tasks/drawing/markings-renderer.cjsx

```diff
@@ -36,7 +35,7 @@ module.exports = React.createClass
   render: ->
     skippedMarks = 0
     <g>
-      {for annotation in @props.classification?.annotations ? []      
+      {for annotation in @props.classification?.annotations ? []
```

Comment:
We should probably change this component to take an array of annotations as a prop. Then I don't think you would have to create a fake classification just to pass in annotations to be rendered.